### DISCUSSION TITLE
The model: ` gpt - 4 ` does not exist

### DISCUSSION THREAD

**Discussion by Zeus64** [AliDehbansiahkarbon/ChatGPTWizard#35] [Q&A]
When i select gpt-4 in the combo I receive The model: ` gpt - 4 ` does not exist
what to do to use gpt 4?

**Comment by Zeus64** [comment]
yes sure, find below this image of the settings :
![image](https://user-images.githubusercontent.com/4325047/232850092-3d446f99-7e17-48a0-b8c7-3a353c97fba1.png)
but it's seam to not work :(

by the way congratulation for your work !

**Comment by AliDehbansiahkarbon** [comment]
Hi, as I know this is not a bug in the plug-in.
Read the documentation that provided by OpenAI, please. 
To use the "gpt-4" model you need to upgrade your plan on OpenAI, it's not available in the free plan.

**Comment by alextroD7** [comment]
Hi, same error, and my token is not from free plain|

**Comment by AliDehbansiahkarbon** [comment]
Ok, I have updated the model list on the settings form, pull the latest version from GitHub and try, please.
It depends totally on your subscription level, it works for me, all the listed models were tested today and work perfectly at my end now.

The latest models list is available here: https://platform.openai.com/docs/guides/text-generation

Tested:

![image](https://github.com/AliDehbansiahkarbon/ChatGPTWizard/assets/5601608/f476c420-753d-4d6f-b4c1-dfee4face31d)
